cmake/cross-toolchain-mingw32.cmake

```diff
@@ -8,7 +8,7 @@ set( CMAKE_CXX_COMPILER i686-w64-mingw32-g++ )
 set( CMAKE_RC_COMPILER i686-w64-mingw32-windres )
 
 # Target prefix
-set( CMAKE_FIND_ROOT_PATH /usr/i686-w64-mingw32 )
+set( CMAKE_FIND_ROOT_PATH /usr/i686-w64-mingw32;${CMAKE_PREFIX_PATH} )
```

Comment:
Surely this doesn't do anything, as the toolchain file is executed before CMakeLists.txt, so CMAKE_PREFIX_PATH wouldn't be set (unless you set it on the command line.)

Reply:
Oh it's for building external_deps, not compiling the game. Could use a comment